docs/src/pages/docs/usage/dates-times.mdx

```diff
@@ -187,6 +187,33 @@ function Component() {
 }
 ```
 
+Furthermore, `relativeTime` will automatically use [`numeric: 'auto'`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/RelativeTimeFormat#numeric) when the time difference divided by the unit is a whole number without a fractional part (e.g. exactly one day). This enables natural phrases like "yesterday" instead of "1 day ago" when appropriate.
+
+You can use utility functions like [`startOfDay`](https://date-fns.org/docs/startOfDay) to ensure that the time difference is a whole number:
+
+```js
+import {useFormatter, useTimeZone} from 'next-intl';
+import {startOfDay} from 'date-fns';
+import {tz} from '@date-fns/tz';
+
+const format = useFormatter();
+const timeZone = useTimeZone();
+
+const now = new Date('2020-12-23T10:36:00.000Z');
+const dateTime = new Date('2020-12-22T08:23:00.000Z');
+
+function normalize(date) {
+  // The "start of a day" depends on a time zone
+  return startOfDay(date, {in: tz(timeZone)});
+}
+
+// Renders "yesterday" instead of "1 day ago"
+const result = format.relativeTime(normalize(dateTime), {
+  now: normalize(now),
+  unit: 'day'
+});
+```
```

Comment:
Been scratching my head a bit over this code snippet. I think it should now be correct, but it's a bit verbose.

Reply:
This is technically what I had to do anyways with my current approach (with the exception of using Luxon instead of date-fns) to ensure that `numeric: auto` gave the proper localized text for "yesterday", "today", etc.

Reply:
Not sure if there's a way to simplify this, unless we allow passing in an optional config param like "shouldDormalizeDate: boolean` to the `format.relativeTime()` options - but that gets into the issue of requiring `next-intl` to supply its own "start of day" function calculation if it wants to be date-library agnostic.